### PR TITLE
GROMACS 5.1.2 combined with PLUMED 2.2.3

### PR DESCRIPTION
**Pull request by victorusu**

### CODE REVIEW

**Comment by victorusu** [comment]
>  eb -f  GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11.eb

== temporary log file in case of crash /tmp/eb-2McgM3/easybuild-7OCUMx.log
== processing EasyBuild easyconfig $HOME/easybuild/devel/tools/easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11.eb
== building and installing GROMACS+PLUMED/5.1.2-2.2.3-CrayGNU-2016.11...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing [skipped]
== installing...
== taking care of extensions...
== postprocessing...
== sanity checking...
== cleaning up...
== creating module...
== permissions...
== packaging...
== COMPLETED: Installation ended successfully
== Results of the build can be found in the log file(s) /apps/dom/UES/6.0.UP02/sandbox-vh/easybuild/software/GROMACS+PLUMED/5.1.2-2.2.3-CrayGNU-2016.11/easybuild/easybuild-GROMACS+PLUMED-5.1.2-2.2.3-20161121.112414.log
== Build succeeded for 1 out of 1
== Temporary log file(s) /tmp/eb-2McgM3/easybuild-7OCUMx.log* have been removed.
== Temporary directory /tmp/eb-2McgM3 has been removed.

> eb -f  GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11-cuda-8.0.eb

== temporary log file in case of crash /tmp/eb-_Yer02/easybuild-mRi4yv.log
== processing EasyBuild easyconfig $HOME/easybuild/devel/tools/easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11-cuda-8.0.eb
== building and installing GROMACS+PLUMED/5.1.2-2.2.3-CrayGNU-2016.11-cuda-8.0...
== fetching files...
== creating build dir, resetting environment...
== unpacking...
== patching...
== preparing...
== configuring...
== building...
== testing [skipped]
== installing...
== taking care of extensions...
== postprocessing...
== sanity checking...
== cleaning up...
== creating module...
== permissions...
== packaging...
== COMPLETED: Installation ended successfully
== Results of the build can be found in the log file(s) /apps/dom/UES/6.0.UP02/sandbox-vh/easybuild/software/GROMACS+PLUMED/5.1.2-2.2.3-CrayGNU-2016.11-cuda-8.0/easybuild/easybuild-GROMACS+PLUMED-5.1.2-2.2.3-20161121.112619.log
== Build succeeded for 1 out of 1
== Temporary log file(s) /tmp/eb-_Yer02/easybuild-mRi4yv.log* have been removed.
== Temporary directory /tmp/eb-_Yer02 has been removed.

**Comment by twrobinson** [comment]
Please double check that it is building GPU-enabled version. Cheers, tim

**Comment by gppezzi** [comment]
@victorusu  where is the .eb for PLUMED?

**Comment by gppezzi** [comment]
found it, it is already merged: 
https://github.com/eth-cscs/tools/blob/master/easybuild/easyconfigs/p/PLUMED/PLUMED-2.2.3-CrayGNU-2016.11.eb

File: easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11-cuda-8.0.eb [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# contributed by Luca Marsella (CSCS)
+#         and Victor Holanda Rusu (CSCS)
+#
+easyblock = 'CMakeMake'
+name = 'GROMACS+PLUMED'
+version = "5.1.2-2.2.3"
+cudaversion = '8.0'
+versionsuffix = '-cuda-%s' % cudaversion
```
Comment (gppezzi) [inline easybuild/easyconfigs/g/GROMACS/GROMACS-5.1.2+PLUMED2.2.3-CrayGNU-2016.11-cuda-8.0.eb]: I'd suggest you add `PLUMED` to the version suffix and remove from the name:
```
name = 'GROMACS'
version = "5.1.2"
versionsuffix = '-PLUMED-2.2.3-cuda-%s' % cudaversion
```
and rename the .eb file accordingly: `GROMACS-5.1.2-CrayGNU-2016.11-PLUMED-2.2.3-cuda-8.0.eb`

**Comment by gppezzi** [comment]
@victorusu please rename the .eb file accordingly in order to match the new version/versionsuffix: `GROMACS-5.1.2-CrayGNU-2016.11-PLUMED-2.2.3-cuda-8.0.eb` and `GROMACS-5.1.2-CrayGNU-2016.11-PLUMED-2.2.3.eb`

**Comment by gppezzi** [comment]
awesome PR, thanks @victorusu

**Pull request merged**